projects/documentation/content/_includes/partials/sidenav.njk

```diff
@@ -51,13 +51,25 @@
         multiLevel
     >
         <span slot="no-js">Tools</span>
-        {% for component in collections['tool-examples'] %}
+        {% for toolRooot in collections['tool-root'] %}
```

Comment:
```suggestion
        {% for toolRoot in collections['tool-root'] %}
```

And subsequent areas, unless this is intential.

Reply:
🙈